pex/resolver.py

```diff
@@ -193,7 +200,11 @@ def resolve(self, resolvables, resolvable_set=None):
         if resolvable in processed_resolvables:
           continue
         packages = self.package_iterator(resolvable, existing=resolvable_set.get(resolvable.name))
-        resolvable_set.merge(resolvable, packages, parent)
+
+        # TODO: Remove blacklist strategy in favor of smart requirement handling
```

Comment:
this TODO is good, but it'd be great to make the temporary nature of this change more evident in the docstring(s) below where developers are more likely to look. Would be good to include the ticket in there as well.